README.md

```diff
@@ -91,6 +92,7 @@ Symbol                              | NET35              | NET40              |
 * `FEATURE_LEGACY_REFLECTION_API` - provides a shim for .NET 3.5/4.0 that emulates the `TypeInfo` API available in .NET 4.5+ and .NET Core.
 * `FEATURE_LISTSORT` - enables support for features that make use of System.ComponentModel.ListSortDescription.
 * `FEATURE_NETCORE_REFLECTION_API` - provides shims to implement missing functionality in .NET Core that has no alternatives.
+* `FEATURE_PEVERIFY` - enables verification of dynamic assemblies using PEVerify during tests. (Only defined on Windows builds since Windows is currently the only platform where PEVerify is available.)
```

Comment:
Since this is a unit test only symbol could we name it `FEATURE_TEST_PEVERIFY` to make that fact clear, which would group it with the other 2 below.

Also each column should have a tick mark rather than a cross.